tests/test_stability.py

```
@@ -0,0 +1,123 @@
+"""画面安定性判定機能のテスト"""
+import pytest
```

Ruff (0.12.2)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)